tests/functional/workloads/ocp/logging/test_openshift-logging.py

```
@@ -45,22 +45,28 @@ class Testopenshiftloggingonocs(E2ETest):
 
     @pytest.fixture()
     def create_pvc_and_deployment_pod(self, request, pvc_factory, pod_factory):
-        """"""
+        """Create PVC and deployment pod for logging test"""
+        logger.info("Setting up PVC and deployment pod for logging test")
 
         def finalizer():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)